boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -8,9 +8,11 @@ Overview
 
 The IT8XXX2 is a 32-bit RISC-V Micro-controller.
 And a highly integrated embedded controller with system functions.
-It is suitable for mobile system applications.
+It is suitable for mobile system applications. The picture below is
+the INTEL_IT81302_81202_Colay_MECC_V01 develop board (also known as
```

Comment:
This name looks weird. Can we spell it without underscores and in a more more human friendly way? Also EVB suggest evaluation board, not development board.

Reply:
OK, the name INTEL_IT81302_81202_Colay_MECC_V01 is printed on the board. We can call it IT81302 MECC board.